js/pages/vocabulary/components/search.js

```diff
@@ -30,330 +30,331 @@ define([
 	commonUtils,
 	vocabularyProvider
 ) {
-	class Search extends AutoBind(Component) {
-		constructor(params) {
-			super(params);
-			this.currentSearch = ko.observable('');
-			this.loading = ko.observable(false);
-			this.domainsLoading = ko.observable(true);
-			this.vocabulariesLoading = ko.observable(true);
-			this.canSearch = ko.observable(true);
-			this.showAdvanced = ko.observable(false);
-			this.domains = ko.observableArray();
-			this.vocabularies = ko.observableArray();
-			this.selected = {
-				domains: new Set(),
-				vocabularies: new Set(),
-			};
-			this.data = ko.observableArray([]);
-			this.searchExecuted = ko.observable(false);
-			this.searchColumns = ko.observableArray([]);
-			this.searchOptions = ko.observable();
+		class Search extends AutoBind(Component) {
+			constructor(params) {
+				super(params);
+				console.log(params);
+				this.currentSearch = ko.observable('');
+				this.loading = ko.observable(false);
+				this.domainsLoading = ko.observable(true);
+				this.vocabulariesLoading = ko.observable(true);
+				this.canSearch = ko.observable(true);
+				this.showAdvanced = ko.observable(false);
+				this.domains = ko.observableArray();
+				this.vocabularies = ko.observableArray();
+				this.selected = {
+					domains: new Set(),
+					vocabularies: new Set(),
+				};
+				this.data = ko.observableArray([]);
+				this.searchExecuted = ko.observable(false);
+				this.searchColumns = ko.observableArray([]);
+				this.searchOptions = ko.observable();
+				this.params = params;
 
-			this.isInProgress = ko.computed(() => {
-				return this.domainsLoading() === true && this.vocabulariesLoading() === true;
-			});
-			this.isInProgress.subscribe((isInProgress) => {
-				if (!isInProgress) {
-					if (params.query()) {
+				this.isInProgress = ko.computed(() => {
+					return this.domainsLoading() === true && this.vocabulariesLoading() === true;
+				});
+				/*
```

Comment:
Why not to remove this if not required?

Reply:
Is it not required?  This wasn't a change in this PR.